go.sum

```diff
@@ -84,6 +84,7 @@ github.com/gogo/protobuf v1.3.1 h1:DqDEcV5aeaTmdFBePNpYsp3FlcVH/2ISVVM9Qf8PSls=
 github.com/gogo/protobuf v1.3.1/go.mod h1:SlYgWuQ5SjCEi6WLHjHCa1yvBfUnHcTbrrZtXPKa29o=
 github.com/golang/glog v0.0.0-20160126235308-23def4e6c14b/go.mod h1:SBH7ygxi8pfUlaOkMMuAQtPIUF8ecWP5IEl/CR7VP2Q=
 github.com/golang/groupcache v0.0.0-20160516000752-02826c3e7903/go.mod h1:cIg4eruTrX1D+g88fzRXU5OdNfaM+9IcxsU14FzY7Hc=
+github.com/golang/groupcache v0.0.0-20190129154638-5b532d6fd5ef h1:veQD95Isof8w9/WXiA+pa3tz3fJXkt5B7QaRBrM62gk=
```

Comment:
🤔 which version of go are you running?

Reply:
1.16.1; you get a different `go mod tidy` result? The CI test passed for that, I presumed that meant this file was as expected.

Reply:
heh, [I think our CI is still on go1.15](https://github.com/vmware-tanzu/sonobuoy/blob/master/.github/workflows/ci-lint.yaml#L18). I thought go1.16 mod is always less noisy because I've only seen go1.16 doing `go mod tidy` to delete entries, and still backwards compatible with 1.15 🤔